scripts/tests/calibnet_stateless_mode_check.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+set -euxo pipefail
+
+# This script tests the stateless mode of a forest node
+
+source "$(dirname "$0")/harness.sh"
+
+forest_init_stateless
+
+echo "Verifying the heaviest tipset to be the genesis"
+HEAD_CID=$($FOREST_CLI_PATH chain head | jq -r '.[0]')
+assert_eq "$HEAD_CID" "bafy2bzacecyaggy24wol5ruvs6qm73gjibs2l2iyhcqmvi7r7a4ph7zx3yqd4"
+
+STATELESS_NODE_ADDRESS=$($FOREST_CLI_PATH net listen | tail -n 1)
+echo "Stateless node address: $STATELESS_NODE_ADDRESS"
+STATELESS_NODE_PEER_ID=$(echo "$STATELESS_NODE_ADDRESS" | cut --delimiter="/" --fields=7 --zero-terminated)
+echo "Stateless node peer id: $STATELESS_NODE_PEER_ID"
+
+# Run a normal forest node that only connects to the stateless node
+CONFIG_PATH="./forest_config.toml"
+cat <<- EOF > $CONFIG_PATH
+	[network]
+	listening_multiaddrs = ["/ip4/127.0.0.1/tcp/0"]
+	bootstrap_peers = ["$STATELESS_NODE_ADDRESS"]
+EOF
+
+# Disable discovery to not connect to more nodes
+$FOREST_PATH --chain calibnet --encrypt-keystore false --auto-download-snapshot --config "$CONFIG_PATH" --rpc false --mdns false --kademlia false --metrics-address 127.0.0.1:6117 &
```

Comment:
Could we disable `mdns` and `kademlia` in config? Unless you think passing it as flags makes more sense.

Reply:
I've no preference, fixed as suggested